### PR TITLE
PIP-704 add preflight md5 option

### PR DESCRIPTION
**Pull request by p-sud**

### CODE REVIEW

File: accession/accession.py [hunk 1]
```diff
@@ -63,12 +74,50 @@ def __init__(self, steps, analysis, connection, lab, award):
...
+    def logger(self) -> logging.Logger:
+        """
+        Creates the instance's logger if it doesn't already exist, then returns the
+        logger instance. Configured to log both to stderr (StreamHandler default) and to
+        a log file.
+        """
+        if self._logger is None:
+            logger = logging.getLogger(__name__)
+            logger.setLevel(logging.DEBUG)
+            formatter = logging.Formatter(
```
Comment (ottojolanki) [inline accession/accession.py]: https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles different ways of formatting should not be mixed and matched. I have never encountered problems, but everyone keeps saying this so we should probably obey..

Comment (p-sud) [reply accession/accession.py]: Well, mixing formatters certainly complicates any potential log parsing. So I guess here we'd want to force the encode utils loggers to use the same formatter?

Comment (ottojolanki) [reply accession/accession.py]: I mean that one should either use % style formatting or { style, but mixing and matching is risky.

Comment (p-sud) [reply accession/accession.py]: Ah gotcha.

Comment (p-sud) [reply accession/accession.py]: So are you proposing any additional changes be made wrt the log formatting?

Comment (p-sud) [reply accession/accession.py]: I updated the logger calls more consistent in formatting, now only using `%s` style formatting within the arguments.

**Pull request merged**